src/main/java/org/candlepin/model/Entitlement.java

```diff
@@ -166,6 +166,14 @@ public String getOwnerId() {
         return (owner == null) ? null : owner.getId();
     }
 
+    /**
+     * @return the owner key of this Entitlement.
+     */
+    @Override
+    public String getOwnerKey() {
+        return this.owner == null ? null : this.owner.getOwnerKey();
```

Comment:
Same deal here; make sure we don't access Hibernate-managed fields directly in the utility methods of our entities.

(note this occurs in more entity classes, but I'm not going to comment on all of them)

Reply:
Updated throughout